src/TestSuite/Fixture/ChecksumTestFixture.php

```diff
@@ -26,15 +26,9 @@ class ChecksumTestFixture extends TestFixture
     protected static $_tableHashes = [];
 
     /**
```

Comment:
these comments are _not_ the same as the parent one - so these should be retained

Reply:
added them back ;)